easybuild/easyconfigs/q/QCG-PilotJob/QCG-PilotJob-0.13.1-gfbf-2022b.eb

```diff
@@ -0,0 +1,55 @@
+easyblock = 'PythonBundle'
+
+name = 'QCG-PilotJob'
+version = '0.13.1'
+
+homepage = 'https://qcg-pilotjob.readthedocs.org'
+description = """A python service for easy execution of many tasks inside a single allocation."""
+
+toolchain = {'name': 'gfbf', 'version': '2022b'}
+
+dependencies = [
+    ('Python', '3.10.8'),
+    ('SciPy-bundle', '2023.02'),
+    ('PyZMQ', '25.1.0'),
+    ('Kaleido', '0.2.1'),
+    ('statsmodels', '0.14.0'),
+    ('plotly.py', '5.13.1'),
+]
+
+use_pip = True
+sanity_pip_check = True
```

Comment:
@SimonPinches I think this is the default now and you can remove it, hence the warning from the bot.

Reply:
Thanks @verdurin, fixed now.